src/robot.coffee

```diff
@@ -237,7 +238,23 @@ class Robot
     @middleware.listener.register middleware
     return undefined
 
-  # Public: Passes the given message to any interested Listeners.
+  # Public: Registers new middleware for execution before matching
+  #
+  # middleware - A function that determines whether or not a given matching
+  #              Listener should be executed. The function is called with
```

Comment:
"A function that determines whether or not Listener matching should commence for a given Message." (or something like that)